ShopAppTests/App/Account/Controllers/OrderListViewControllerSpec.swift

```diff
@@ -56,13 +58,24 @@ class OrderListViewControllerSpec: QuickSpec {
                 disposeBag = DisposeBag()
             }
             
-            it("should have correct tableView sections counts") {
+            it("should have tableView without data") {
                 viewController.viewModel.items.asObservable()
                     .subscribe(onNext: { items in
-                        expect(viewController.tableView.numberOfSections) == items.count
+                        expect(viewController.tableView.numberOfSections) == 0
                     })
                 .disposed(by: disposeBag)
             }
+            
+            it("should have tableView with data") {
+                viewModel.isNeedToReturnData = true
+                viewModel.reloadData()
+                
+                viewController.viewModel.items.asObservable()
+                    .subscribe(onNext: { items in
+                        expect(viewController.tableView.numberOfSections) == items.count
```

Comment:
Also you need to test:
- `strongSelf.stopLoadAnimating()`
- `strongSelf.tableProvider.orders = orders` like in details controller